cleaned_data/loganXML/20220114_Braddock_Paige_edited.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mods xmlns="http://www.loc.gov/mods/v3"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xs="http://www.w3.org/2001/XMLSchema"
+      xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+   <identifier type="local">20220114_Braddock_Paige_edited</identifier>
+   <titleInfo>
+      <title>Interview with Paige Braddock, 2022-01-14</title>
+   </titleInfo>
+   <abstract>Interview with Paige Braddock, Chief Creative Officer for Charles Shulz's studio and former East Tennessee resident, conducted by Casey Kaufman. Braddock discusses her time in East Tennessee and relates her California wildfire experiences to the wildfire in Gatlinburg. She explains how her experiences have helped her create artwork for the Gatlinburg wildfire. Conducted over Zoom due to the COVID-19 pandemic. This is a representative quote from the interview: "If there is a story or an event or something that is almost impossible to illustrate with a photograph, sometimes a drawing or illustration can get to the heart of that so that you're not so much showing a representation of something that happened, but you're getting at the emotion behind what happened."</abstract>
+   <name>
```

Comment:
@logantillman - thanks so much for these records. Can you alter the name field here to the following:

```
<name authority="naf" valueURI="http://id.loc.gov/authorities/names/n2002022941">
      <namePart>Braddock, Paige</namePart>
      <role>
         <roleTerm authority="marcrelator"
                   valueURI="http://id.loc.gov/vocabulary/relators/ive">Interviewee</roleTerm>
      </role>
</name>
```
We don't always have authorities established in https://id.loc.gov, but we use them when we do.